static/css/components/buttonCta--js.less

```diff
@@ -0,0 +1,51 @@
+/**
+ * CtaBtn
+ * https://github.com/internetarchive/openlibrary/wiki/Design-Pattern-Library#ctabtn
+ * This stylesheet will only apply if JS is present on the page.
+ */
+ .cta-btn,
+ // override default anchor styles
+ a.cta-btn {
+   font-size: 14px;
+   color: @white !important;
+   width: 100%;
+   display: block;
+   border: 0;
+   border-radius: 5px;
+   box-sizing: border-box;
+   cursor: pointer;
+   text-decoration: none;
+   font-family: @lucida_sans_serif-1;
+   text-align: center;
+   padding: 7px 20px;
+   white-space: nowrap;
+   background-color: @mid-grey;
+   line-height: 1.5em;
+ 
+   &:hover {
+     text-decoration: underline;
+   }
+   &--available {
+     background-color: @primary-blue;
+     &--load {
+       background-color: @primary-blue !important;
+       background: url(../../../static/images/indicator.gif) center center no-repeat;
+       opacity: 0.6;
+     }
+   }
+   &--unavailable {
+     background-color: @orange;
+     &--load {
+       background-color: @orange !important;
```

Comment:
I think only the --load rules should be needed in this file!

Reply:
My bad!
Didn't realize that sooner.
Changes have been made.